src/renderers/dom/client/eventPlugins/ChangeEventPlugin.js

```diff
@@ -176,7 +186,11 @@ function startWatchingForValueChange(target, targetID) {
   );
 
   Object.defineProperty(activeElement, 'value', newValueProp);
-  activeElement.attachEvent('onpropertychange', handlePropertyChange);
+  if (activeElement.attachEvent) {
+    activeElement.attachEvent('onpropertychange', handlePropertyChange);
+  } else {
+    activeElement.addEventListener('propertychange', handlePropertyChange, false);
```

Comment:
Ya I saw that as well, but right below that it also demonstrates it using `addEventListener`, and it does seem to work in my testing. It could probably stand further testing